src/constructN.js

```diff
@@ -33,12 +31,22 @@ var nAry = require('./nAry');
  *      R.map(R.constructN(1, Widget), allConfigs); // a list of Widgets
  */
 module.exports = _curry2(function constructN(n, Fn) {
-    var f = function() {
-        var Temp = function() {}, inst, ret;
-        Temp.prototype = Fn.prototype;
-        inst = new Temp();
-        ret = Fn.apply(inst, arguments);
-        return Object(ret) === ret ? ret : inst;
-    };
-    return n > 1 ? curry(nAry(n, f)) : f;
```

Comment:
Ah, yes. I'll add a test case for a zero-arg constructor and reinstate the special case.